### PR TITLE
t/119 - generated content

### PR DESCRIPTION
**Pull request by usqr**
Script generated content.

### CODE REVIEW

File: user-interface-balloon-block/index.html [hunk 1]
```diff
@@ -4,66 +4,103 @@
...
+
+				<figure class="image image-style-side" height="400">
+					<img
+						src="/assets/images/ckdemo/editor-types/tajmahal.jpg"
+						srcset="
+							/assets/images/ckdemo/editor-types/tajmahal.jpg,
+							/assets/images/ckdemo/editor-types/tajmahal_2x.jpg 2x
+						"
+						alt="Taj Mahal illustration."
+					/>
```
Comment (Dumluregn) [inline user-interface-balloon-block/index.html:31]: `/assets/images/ckdemo` path is incorrect and such images are broken (there are some others in the PR). We should use the absolute paths everywhere (only then they can be the same in the website and here).

File: markdown/index.html [hunk 2]
```diff
@@ -19,16 +19,16 @@ <h1>Markdown editor</h1>
 	</div>
 	<div class="editor-wrapper">
 		<div id="cke5-markdown-demo">
-## Markdown output 🛫
+			## Markdown output 🛫
```
Comment (Dumluregn) [inline markdown/index.html:22]: If Markdown code is indented, the editor thinks it's a codeblock:

image

If should be outdented like the rest of the content in this demo.

File: user-interface-document/index.html [hunk 3]
```diff
@@ -4,97 +4,150 @@
...
+									</tr>
+									<tr>
+										<td>5:00 PM - 8:00 PM</td>
+										<td>
+											<strong
+												>Tuscan vineyards at a
+												glance</strong
+											>
+											- wine-tasting&nbsp;<br />with
+											Frederico Riscoli
```
Comment (Dumluregn) [inline user-interface-document/index.html:101]: Is there _anything_ we could change in configuration to make the line splitting look a bit better? If anyone takes a look at it, it seems broken at the first glance 😞

**Review by Dumluregn — CHANGES_REQUESTED** [review]

File: feature-rich/index.html [hunk 4]
```diff
@@ -4,159 +4,208 @@
 For licensing, see LICENSE.md or http://ckeditor.com/license
 -->
 <html lang="en">
+	<head>
+		<meta charset="utf-8" />
```
[the remark this answers is not in the document]
Comment (usqr) [reply feature-rich/index.html:8]: It's a prettier default. 🤷🏻

File: user-interface-balloon-block/index.html (hunk 1 above)
Comment (usqr) [reply user-interface-balloon-block/index.html]: fixed, all the incorect paths

File: user-interface-document/index.html (hunk 3 above)
Comment (usqr) [reply user-interface-document/index.html]: done

File: headless/index.jsx [hunk 5]
```diff
@@ -54,9 +54,10 @@ function App() {
 						<li>
 							Or your own design system 💪
 						</li>
-					</ul>"
-				onReady={ ( editor ) => {
-					setEditor( editor );
+					</ul>;
```
Comment (Dumluregn) [inline headless/index.jsx:57]: `;` stays in the editor content:
image

File: markdown/index.html (hunk 2 above)
Comment (Dumluregn) [reply markdown/index.html:22]: I can still reproduce this issue:
image

File: user-interface-balloon/index.html [hunk 6]
```diff
@@ -4,63 +4,69 @@
...
+				</figure>
+
+				<p>
+					If you enjoyed my previous articles in which we discussed wandering around
+					<a href="https://en.wikipedia.org/wiki/Copenhagen">Copenhagen</a>
+					and
+					<a href="https://en.wikipedia.org/wiki/Vilnius">Vilnius</a>
+					, you’ll definitely love exploring
+					<a href="https://en.wikipedia.org/wiki/Warsaw">Warsaw</a>
+					.
```
Comment (Dumluregn) [inline user-interface-balloon/index.html:34]: Not a hard blocker, but due to this formatting we have extra whitespace before comma and dot:
image

File: user-interface-bottom-toolbar/index.html [hunk 7]
```diff
@@ -1,63 +1,72 @@
+export default (content) => `
```
Comment (Dumluregn) [inline user-interface-bottom-toolbar/index.html:1]: Some artifact.

**Review by Dumluregn — CHANGES_REQUESTED** [review]

**Pull request merged**